CRM/Event/Tokens.php

```diff
@@ -135,8 +206,20 @@ protected function getEventTokenValues(int $eventID = NULL): array {
       $tokens['end_date']['text/html'] = !empty($event['end_date']) ? new DateTime($event['end_date']) : '';
       $tokens['event_type_id:label']['text/html'] = CRM_Core_PseudoConstant::getLabel('CRM_Event_BAO_Event', 'event_type_id', $event['event_type_id']);
       $tokens['event_type_id:name']['text/html'] = CRM_Core_PseudoConstant::getName('CRM_Event_BAO_Event', 'event_type_id', $event['event_type_id']);
+      $tokens['contact_email']['text/html'] = $event['loc_block_id.email_id.email'];
+      $tokens['contact_email_2']['text/html'] = $event['loc_block_id.email_2_id.email'];
       $tokens['contact_phone']['text/html'] = $event['loc_block_id.phone_id.phone'];
+      $tokens['contact_phone_type_id:label']['text/html'] = $event['loc_block_id.phone_id.phone_type_id:label'];
+      $tokens['contact_phone_type_id']['text/html'] = $event['loc_block_id.phone_id.phone_type_id'];
+      $tokens['contact_phone_ext']['text/html'] = $event['loc_block_id.phone_id.phone_ext'];
+      $tokens['contact_phone_2']['text/html'] = $event['loc_block_id.phone_2_id.phone'];
+      $tokens['contact_phone_type_id_2:label']['text/html'] = $event['loc_block_id.phone_2_id.phone_type_id:label'];
+      $tokens['contact_phone_type_id_2']['text/html'] = $event['loc_block_id.phone_2_id.phone_type_id'];
+      $tokens['contact_phone_ext_2']['text/html'] = $event['loc_block_id.phone_2_id.phone_ext'];
       $tokens['contact_email']['text/html'] = $event['loc_block_id.email_id.email'];
+      $tokens['confirm_email_text']['text/plain'] = $event['confirm_email_text'];
```

Comment:
Is there a reason why these 3 are text/plain rather than text/html?

Reply:
I can kind of now looking at the ManageEvent get confirm_email_text and pay_later_text as those are text areas but pay_later_receipt is a wysiwyg https://github.com/civicrm/civicrm-core/blob/master/CRM/Event/Form/ManageEvent/Fee.php#L265 so should be text/html IMO

Reply:
Ah great that you should ask ... there is!

If the value is stored as text, which this is, then specifying text/plain here means it gets converted to html in the token processor if rendering in html format.